.github/ISSUE_TEMPLATE/SUPPORT_QUESTION.md

```diff
@@ -0,0 +1,20 @@
+---
+name: 💬 Support Question
+about: If you have a question, please check out our Slack or StackOverflow!
+
+---
+
+--------------^ Click "Preview" for a nicer view!
+
+Apache Cordova primarily use GitHub as an issue tracker; for usage and support questions, please check out these resources below. Thanks.
```

Comment:
I'd suggest the following

> Apache Cordova uses GitHub Issues as an issue tracker *only*.
For usage and support questions, please check out the resources below. Thanks!

Reply:
Good point, much better wording.

Used:
```
Apache Cordova uses GitHub Issues as a feature request and bug tracker _only_.
For usage and support questions, please check out the resources below. Thanks!
```